docs/querying/granularities.md

```diff
@@ -38,10 +38,27 @@ You can specify a time period as a [simple](#simple-granularities) string, as a
 
 Simple granularities are specified as a string and bucket timestamps by their UTC time (e.g., days start at 00:00 UTC).
 
-Supported granularity strings are: `all`, `none`, `second`, `minute`, `fifteen_minute`, `thirty_minute`, `hour`, `day`, `week`, `month`, `quarter` and `year`.
-
-* `all` buckets everything into a single bucket
-* `none` does not bucket data (it actually uses the granularity of the index - minimum here is `none` which means millisecond granularity). Using `none` in a [TimeseriesQuery](../querying/timeseriesquery.md) is currently not recommended (the system will try to generate 0 values for all milliseconds that didn’t exist, which is often a lot).
+Supported granularity strings are: 
+  - `all`
+  - `none`
+  - `second`
+  - `minute`
+  - `five_minute`
+  - `ten_minute`
+  - `fifteen_minute`
+  - `thirty_minute`
+  - `hour`
+  - `six_hour`
+  - `eight_hour`
+  - `day`
+  - `week`
+  - `month`
+  - `quarter` 
+  - `year`
+
+> NOTE
+> * `all` buckets everything into a single bucket
```

Comment:
The basic behaviour of `all` and `none` should probably not be a side-note. 

We could maybe break this into two parts:
```
The supported simple granularity strings are as under. Each of these divide data into buckets of the respective size.
- `second`
- `minute`
- ...
- `year`

Special granularity strings:
- `all`: Puts everything into a single bucket
- `none`: Does not bucket data at all. Instead, it uses the granularity ...
```

It makes sense to make the following part a side note though:
```
Using `none` in a [TimeseriesQuery](../querying/timeseriesquery.md) is currently not recommended. This is because the system will try to generate 0 values for all milliseconds that didn’t exist, which is often a lot.
```

Reply:
There appears to be a related change in #12712

FYI @vtlim

Reply:
@kfaraz Thanks for your suggestion. I adopted @vtlim 's suggestion to address your comment.